### ISSUE TITLE
geom_sina omits data when there's only one point

### ISSUE THREAD

**Issue by Raisinscone** [thomasp85/ggforce#51]
I'm having an issue with geom_sina (a package which I love, by the way) where if there is only one value that value is omitted.  Here is a reproducible example:

```
simple <- data.frame(ID = c(rep(1,7)),
                     group = c("alpha", "beta", "alpha", "beta", "alpha", "beta", "alpha"),
                     version = c("on", "on", "off", "off", "on", "on", "off"),
                     value = c(1:4, 1:3))

ggplot() +   
    geom_point(data = simple, aes(x = ID, y = value, colour = ID),
               position = position_jitter(w = 0.2, h = 0)) +
    theme(legend.position = "none",
          axis.title.y = element_blank()) +
    facet_grid(version ~ group, shrink=FALSE)

ggplot() +   
    geom_sina(data = simple, aes(x = ID, y = value, colour = ID)) +
    theme(legend.position = "none",
          axis.title.y = element_blank()) +
    facet_grid(version ~ group, shrink=FALSE)
```

That creates two plots.  The geom_point plot has all points, but the geom_sina one has omitted the single point in 'beta' x 'off' and gives the following warning:

> Warning message:
> Removed 1 rows containing missing values (geom_sina).

I suspect the removed row is the single point I wanted plotted.  Is there a workaround or fix?

Thanks!

(Edited to fix sample data set)

**Comment by sidiropoulos** [comment]
@Raisinscone this was indeed a bug! I just pushed a fix.

Thanks for reporting!

**Comment by thomasp85** [comment]
Fixed in #52

**Comment by Raisinscone** [comment]
Thanks!  Glad to have been of help.
